builds/e2e/longhaul.yaml

```diff
@@ -226,6 +226,7 @@ jobs:
       name: $(pool.name)
       demands:
         - run-long-haul -equals true
+        - agent.OSName -equals WinPro_x64
```

Comment:
nit: suggest to rename as agent-os-name; so it is obvious that this is not auto-generated capability.

Reply:
And try to have the same order as other windows demands section.

Reply:
Fixed!